From f56503b6d6fac468b691951fee33cae94e938169 Mon Sep 17 00:00:00 2001
From: Lin Huang <hl@rock-chips.com>
Date: Sun, 7 Oct 2018 16:18:37 +0800
Subject: [PATCH] arm64: dts: rockchip: rk1808-evb: fix some wifi related
 setting

with these fixs, AP6212 can work on rk1808-evb board.

Change-Id: Icfdb6dedfcb139debad299852e87586ce0d31de6
Signed-off-by: Lin Huang <hl@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
index 7ec7e1b79478..6e72d4a9a5fb 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
@@ -155,7 +155,7 @@
 		 * - SDIO_RESET_L_WL_REG_ON
 		 * - PDN (power down when low)
 		 */
-		reset-gpios = <&gpio4 RK_PC0 GPIO_ACTIVE_HIGH>;
+		reset-gpios = <&gpio4 RK_PC0 GPIO_ACTIVE_LOW>;
 	};
 
 	vcc5v0_sys: vcc5v0-sys {
@@ -184,7 +184,7 @@
 		pinctrl-names = "default";
 		pinctrl-0 = <&wifi_wake_host>;
 		wifi_chip_type = "ap6212";
-		WIFI,host_wake_irq = <&gpio4 RK_PC1 GPIO_ACTIVE_LOW>;
+		WIFI,host_wake_irq = <&gpio4 RK_PC1 GPIO_ACTIVE_HIGH>;
 		status = "okay";
 	};
 };
@@ -561,10 +561,11 @@
 	cap-mmc-highspeed;
 	cap-sd-highspeed;
 	supports-sdio;
-	sd-uhs-sdr25;
-	sd-uhs-sdr50;
+	keep-power-in-suspend;
+	non-removable;
+	mmc-pwrseq = <&sdio_pwrseq>;
 	sd-uhs-sdr104;
-	status = "disabled";
+	status = "okay";
 };
 
 &sdmmc {
-- 
2.35.3

